include/rapidcheck/Gen.hpp

```diff
@@ -71,13 +71,6 @@ Shrinkable<T> Gen<T>::operator()(const Random &random, int size) const
     auto exception = std::current_exception();
     return shrinkable::lambda([=]() -> T {
       std::rethrow_exception(exception);
-
-      // MSVC HACK: the following is required for MSVC to stop complaining
-      //            about missing return value. Will never be reached, of
-      //            course.
-#ifdef _MSC_VER
-      throw nullptr;
-#endif // _MSC_VER
```

Comment:
Maybe its no longer needed with update 3 to VS, there's lots of C++ standard conformance fixes in the update. With the update the above code generates an "unreachable code" warning. Is it worth figuring out how to tell the difference between VS2015 pre and post update 3?